interface/frontend/src/components/common/OpenDICOM.vue

```diff
@@ -128,7 +129,7 @@
         if (_.isEmpty(dicom)) return ''
         else {
           cornerstone.registerImageLoader(this.view.type, () => {
-            return new Promise((resolve) => { resolve(dicom) })
+            return {promise: new Promise((resolve) => { resolve(dicom) })}
```

Comment:
How come?

Reply:
As far as I can see, the most recent version of the tool expects the loader to return an object with two properties: the promise and cancel function. [Here is](https://github.com/cornerstonejs/cornerstoneTools/blob/29182180ed3f850ba41c609b98b96464affca5f0/examples/exampleTextImageLoader.js#L58) an example.